### PR TITLE
Release 1.0.2

### PR DESCRIPTION
**Pull request by Blackbaud-TrevorBurch**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+# 1.0.2 (2019-08-08)
+
+- Fixed the confirm demo's alert to clearly show the difference in the action taken and button text. [#6](https://github.com/blackbaud/skyux2-demos/pull/6)
```
Comment (Blackbaud-DeniseCisneros) [inline CHANGELOG.md]: Fixed the confirm component's demo to clearly show the difference between the action taken and button text in the alert.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,9 @@
...
+- Fixed the repeater demo's context menu to have a true context menu and not a dropdown menu. [#8](https://github.com/blackbaud/skyux2-demos/pull/8)
```
Comment (Blackbaud-DeniseCisneros) [inline CHANGELOG.md]: Fixed the repeater component's demo to use a context menu and not a dropdown menu.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,9 @@
...
+- Fixed the grid and list view grid demos to only show the `id` or `field` property on each column. [#9](https://github.com/blackbaud/skyux2-demos/pull/9)
```
Comment (Blackbaud-DeniseCisneros) [inline CHANGELOG.md]: Fixed the grid component and list view grid component demos to only show the `id` or `field` property on each column.

**Review by Blackbaud-DeniseCisneros — COMMENTED** [review]
Small tweaks

**Review by Blackbaud-AlexKingman — DISMISSED** [review]

**Review by Blackbaud-DeniseCisneros — APPROVED** [review]
👍

**Pull request merged**